app/modules/sightings/models.py

```diff
@@ -383,7 +388,7 @@ def check_all_job_status(self, all_scheduled):
                     f'{self.guid} is identifying but no identification jobs are running, '
                     'assuming Celery error and starting them all again'
                 )
-            self.ia_pipeline(background=False)
+            self.send_all_identification()
```

Comment:
called from within celery context so just use the method in celery context to retry non started jobs